arrow/arrow.py

```diff
@@ -692,6 +692,25 @@ def timestamp(self):
 
         """
 
+        warnings.warn(
+            "For compatibility with the datetime.timestamp() method this property will removed in the 1.0.0 release, "
```

Comment:
Should we instead say "this property will be replaced with a function in 1.0.0"?